optaplanner-persistence/optaplanner-persistence-jpa/pom.xml

```diff
@@ -34,6 +34,10 @@
       <scope>test</scope>
     </dependency>
     <dependency>
+      <groupId>org.kie</groupId>
+      <artifactId>kie-test-util</artifactId>
```

Comment:
Where's the github repository that contains the source code of this dependency?
Just want to double check something.